test/test.ts

```diff
@@ -88,6 +88,187 @@ describe('linkinator', () => {
     );
   });
 
+  it('should detect relative urls with root relative base', async () => {
+    const scope = nock('http://fake.local')
+      .get('/pageBase/index')
+      .reply(
+        200,
+        `<html>
```

Comment:
Could we move this to a file in the fixtures directory, and use the reply with File feature of nock?

Reply:
(here and below)

Reply:
Certainly. I'll try it tomorrow

Reply:
Done. Also I merged similar test cases with relative base.